.gitignore

```diff
@@ -530,3 +530,6 @@ Client/imgui.ini
 
 # linux build
 Client/linux/sonyheadphonesclient.build/
+
+# Other
+.clang-format
```

Comment:
Do you want this `.clang-format` file?

Reply:
I've never used it, but it could be nice. So sure